lib/cli_common/cli_common/phabricator.py

```diff
@@ -96,7 +96,10 @@ def _clean(diff):
                 ref['type']: ref
                 for ref in diff['refs']
             }
-            diff['baseRevision'] = diff['refs']['base']['identifier']
+            try:
+                diff['baseRevision'] = diff['refs']['base']['identifier']
+            except KeyError:
+                diff['baseRevision'] = None
```

Comment:
I would not ignore the error always, but make sure the first revision in the stack does have a base revision.